### PR TITLE
use hardhat instead of truffle in setup.md (UMA v2.x)

### PR DESCRIPTION
**Pull request by thorstenhirsch**
also use npx for running commands in the project's environment instead of yarn

### CODE REVIEW

File: docs/developers/setup.md [hunk 1]
```diff
@@ -68,13 +68,13 @@ these networks, you'll generally need to know:
...
-The `--network` parameter that's passed to all truffle commands depends on both of these factors. Here's an example:
+The `--network` parameter that's passed to all hardhat commands depends on both of these factors. Here's an example:
 
 ```bash
-yarn truffle console --network rinkeby_mnemonic
+npx hardhat console --network rinkeby_mnemonic
 ```
 
-That command will tell truffle that the user wants to use the Rinkeby testnet and their private key is a mnemonic, or
+That command will tell hardhat that the user wants to use the Rinkeby testnet and their private key is a mnemonic, or
```
Comment (abg4) [inline docs/developers/setup.md:77]: I think we need to take out the private key and mnemonic part of this sentence and change the network argument to be structured as `--network [NETWORK_NAME]`

File: docs/developers/setup.md [hunk 2]
```diff
@@ -93,11 +93,11 @@ With a real mnemonic, this would look like:
 export MNEMONIC="candy maple cake sugar pudding cream honey rich smooth crumble sweet treat"
 ```
 
-Once you've done that you're ready to run a truffle command. When using a mnemonic, your network argument should look
+Once you've done that you're ready to run a hardhat command. When using a mnemonic, your network argument should look
 like `--network [NETWORK_NAME]_mnemonic`. So, for example, using a mnemonic on Kovan would look like:
```
Comment (abg4) [inline docs/developers/setup.md:97]:
```suggestion
like `--network [NETWORK_NAME]`. So, for example, using a mnemonic on Kovan would look like:
```

File: docs/developers/setup.md [hunk 3]
```diff
@@ -93,11 +93,11 @@ With a real mnemonic, this would look like:
...
 
 ```bash
-yarn truffle console --network kovan_mnemonic
+npx hardhat console --network kovan_mnemonic
```
Comment (abg4) [inline docs/developers/setup.md:100]:
```suggestion
npx hardhat console --network kovan
```

File: docs/developers/setup.md [hunk 4]
```diff
@@ -93,11 +93,11 @@ With a real mnemonic, this would look like:
...
 ```
 
 ### Hardware wallets (more secure)
```
Comment (abg4) [inline docs/developers/setup.md:103]: I think we should consider removing the hardware wallet section since these commands do not work with hardhat.

**Comment by smb2796** [comment]
Hey @thorstenhirsch sorry for the delay here and thank you for the contribution.

This was fixed as part of #282, so I am going to close this pr.

**Pull request closed without merging**